src/services_models/services_constructor.jl

```diff
@@ -294,13 +294,16 @@ function construct_service!(
     devices_template::Dict{Symbol, DeviceModel},
     ::Set{<:DataType},
 ) where {S <: PSY.AGC, T <: AbstractAGCFormulation}
-    name = get_service_name(model)
-    service = PSY.get_component(S, sys, name)
-    agc_area = PSY.get_area(service)
+    services = PSY.get_components(S, sys)
+    agc_areas = PSY.get_area.(services)
     areas = PSY.get_components(PSY.Area, sys)
     for area in areas
-        if area != agc_area
-            #    throw(IS.ConflictingInputsError("All area most have an AGC service assigned in order to model the System's Frequency regulation"))
+        if area ∉ agc_areas
```

Comment:
In case you care, a setdiff would be more performant.

Reply:
Fixed